src/UI.java

```diff
@@ -31,8 +32,8 @@ public void displayCoffees() {
         int coffeeIndex = 0;
         Iterator iterator = coffees.iterator();
 
-        while(iterator.hasNext()) {
-            Beverage coffee = (Beverage)iterator.next();
+        while (iterator.hasNext()) {
+            Beverage coffee = (Beverage) iterator.next();
             ++coffeeIndex;
             System.out.printf("%1$d.\t\t%2$s\t\t$%3$,.2f\n", coffeeIndex, coffee.getDescription(), coffee.getPrice());
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.